packages/dialtone-vue2/components/avatar/avatar.mdx

```diff
@@ -1,110 +1,10 @@
-import { Canvas, Story, Subtitle, Controls, Meta } from '@storybook/blocks';
+import { Subtitle, Meta } from '@storybook/blocks';
 import * as AvatarStories from './avatar.stories.js';
 
 <Meta of={AvatarStories}/>
 
 # Avatar
 
 <Subtitle>
-  An avatar is a visual representation of a user or object.
+  Check the documentation [here](https://dialtone.dialpad.com/components/avatar.html).
```

Comment:
Maybe a bit more info about why we did this since this is a point of confusion for many users.

"Please see the component documentation here: [LINK]

Component documentation has been moved to our main documentation site and storybook remains to be used as a component playground only."